src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj

```diff
@@ -107,6 +107,10 @@
 		<!--#if (useMauiPackageReference)-->
 		<PackageReference Include="Microsoft.Maui.Controls" />
 		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" />
+		<!--#else -->
+		<!--#if (mauiEmbedding)-->
+		<PackageReference Include="Microsoft.Maui.Graphics" VersionOverride="$(MauiVersion)" />
```

Comment:
Shouldn't we have a single version of Maui in Directory.Packages.props, and not need to have any kind of version overrides?

Reply:
If only - this is a workaround because Maui uses sdk/workload version for various libraries